src/components/SideBar.vue

```diff
@@ -91,7 +104,10 @@ export default {
      */
     tabs: {
       type: Array,
-      default: () => [{ title: 'flatmap', id: 1 }],
+      default: () => [
+        { title: 'Search', id: 1 },
+        { title: 'Connectivity', id: 2 }
+      ],
```

Comment:
Can we add a type (with the value being search or connectivity) field which would make it clearer and will probably make the logic simpler now and in the future if we decide to have multiple search/connectivity tabs.

Reply:
Looking deeper into the codes, adding a type field will help with the logic at multiple places.

Reply:
Tab type is added and it also supports multiple search tabs and multiple connectivity tabs.